Source/WebKit/UIProcess/WebContextMenuProxy.cpp

```diff
@@ -55,9 +55,13 @@ Vector<Ref<WebContextMenuItem>> WebContextMenuProxy::proposedItems() const
 void WebContextMenuProxy::show()
 {
     ASSERT(m_context.webHitTestResultData());
-    
+
```

Comment:
Weird of Github to show this

Reply:
There was actually some whitespace before!